tests/platform_tests/api/test_sfp.py

```diff
@@ -376,14 +383,14 @@ def test_get_transceiver_info(self, duthosts, enum_rand_one_per_hwsku_hostname,
                     if duthost.sonic_release in ["201811", "201911", "202012", "202106", "202111"]:
                         UPDATED_EXPECTED_XCVR_INFO_KEYS = [
                             key if key != 'vendor_rev' else 'hardware_rev' for key in self.EXPECTED_XCVR_INFO_KEYS]
-                        # self.EXPECTED_XCVR_INFO_KEYS = EXPECTED_XCVR_INFO_KEYS
                     else:
-
-                        if info_dict["type_abbrv_name"] in ["QSFP-DD", "OSFP-8X"]:
-                            UPDATED_EXPECTED_XCVR_INFO_KEYS = self.EXPECTED_XCVR_INFO_KEYS + \
-                                                           self.EXPECTED_XCVR_NEW_QSFP_DD_OSFP_INFO_KEYS + \
-                                                           ["active_apsel_hostlane{}".format(n)
-                                                            for n in range(1, info_dict['host_lane_count'] + 1)]
+                        if info_dict["type_abbrv_name"] == "QSFP-DD":
```

Comment:
@mannytaheri this check is removing the previous check supporting OSFP modules. Can you fix this for both QSFP-DD and OSFP?